clusterous/cluster.py

```diff
@@ -643,6 +640,161 @@ def get_cluster_spec(self):
 
         return spec
 
+    def _nat_ssh_port_forwarding(self, nat_public_ip, controller_private_ip):
+        time.sleep(45)
```

Comment:
Again, there shouldn't be a sleep here. If the goal is to wait for SSH to start, it's best to put this in an Ansible script itself, since that's anyway how we configure all the other instances.